docs/using/installing/index.rst

```diff
@@ -219,11 +229,11 @@ Using Rackspace
 Another way to get a Flocker cluster running is to use Rackspace.
 You'll probably want to setup at least two nodes.
 
-#. Create a new Cloud Server running Fedora 20
+#. Create a new Cloud Server
 
    * Visit https://mycloud.rackspace.com
    * Click "Create Server".
-   * Choose the Fedora 20 Linux distribution as your image.
+   * Choose a supported Linux distribution as your image (for example, Ubuntu 14.04 or CentOS 7).
```

Comment:
Yes, we mulled over this -- until we have an autogenerated list of supported platforms this will have to do.